paymentrequest/payment-handler/index.html

```diff
@@ -1,18 +1,17 @@
 ---
-feature_name: PaymentRequest Credit Cards
-chrome_version: 53
-feature_id: 5639348045217792
+feature_name: Payment Handler API
+chrome_version: 68
+feature_id: 5160285237149696
 ---
 
 <h3>Background</h3>
 <p>
   <a href="https://github.com/w3c/browser-payment-api">PaymentRequest</a> lets
-  you accept payment from different payment methods.
+  you accept payment with a URL-based payment method.
```

Comment:
We still support non-URL payment methods such as "secure-payment-confirmation", so please leave the original text as it was here.